### ISSUE TITLE
Numpad number keysyms do not not work in riverctl.

### ISSUE THREAD

**Issue by Bugbert** [riverwm/river#962]
Steps to reproduce:
Map any numpad key `KP_<number>` to anything
`riverctl map normal None KP_4 spawn alacritty`
The command it is mapped to will not be executed

Note: Other numpad keys like `KP_End` function normally.

**Comment by ifreund** [comment]
It could be that your keyboard uses the Numlock modifier for `KP_<number>` keysyms.

Does `riverctl map normal Mod2 KP_4 spawn alacritty` work?

If not, could you please provide the output of `wev`, `xkbcli interactive-wayland` or similar including your numpad key presses?

**Comment by Bugbert** [comment]
The suggested solution didn't seem to work; here's the output from wev:
```
[07:         wl_seat] name: default
[07:         wl_seat] capabilities: pointer keyboard
[10:    xdg_toplevel] configure: width: 0; height: 0
[09:     xdg_surface] configure: serial: 553
[14:     wl_keyboard] keymap: format: 1 (xkb v1), size: 64756
[14:     wl_keyboard] repeat_info: rate: 50 keys/sec; delay: 300 ms
[14:     wl_keyboard] enter: serial: 557; surface: 3
'                     sym: Return       (65293), utf8: '
[14:     wl_keyboard] modifiers: serial: 0; group: 16
                      depressed: 00000000
                      latched: 00000000
                      locked: 00000010: Mod2
[11:  wl_data_device] data_offer: id: 4278190080
[4278190080:   wl_data_offer] offer: mime_type: text/html
[4278190080:   wl_data_offer] offer: mime_type: text/plain
[4278190080:   wl_data_offer] offer: mime_type: text/plain;charset=utf-8
[4278190080:   wl_data_offer] offer: mime_type: text/plain
[11:  wl_data_device] selection: id: 4278190080
[10:    xdg_toplevel] configure: width: 640; height: 480
                      activated tiled-left tiled-right tiled-top tiled-bottom
[09:     xdg_surface] configure: serial: 554
[10:    xdg_toplevel] configure: width: 1128; height: 1017
                      activated tiled-left tiled-right tiled-top tiled-bottom
[09:     xdg_surface] configure: serial: 560
[13:      wl_pointer] enter: serial: 563; surface: 3, x, y: 865.675781, 533.300781
[13:      wl_pointer] frame
[14:     wl_keyboard] key: serial: 564; time: 298784; key: 36; state: 0 (released)
                      sym: Return       (65293), utf8: ''
[14:     wl_keyboard] key: serial: 565; time: 300464; key: 83; state: 1 (pressed)
                      sym: KP_4         (65460), utf8: '4'
[14:     wl_keyboard] key: serial: 566; time: 300528; key: 83; state: 0 (released)
                      sym: KP_4         (65460), utf8: ''
[14:     wl_keyboard] key: serial: 567; time: 301631; key: 133; state: 1 (pressed)
                      sym: Super_L      (65515), utf8: ''
[14:     wl_keyboard] modifiers: serial: 0; group: 16
                      depressed: 00000040: Mod4
                      latched: 00000000
                      locked: 00000010: Mod2
[10:    xdg_toplevel] close
```
This is all that was printed when the key was pressed:
```
[14:     wl_keyboard] key: serial: 565; time: 300464; key: 83; state: 1 (pressed)
                      sym: KP_4         (65460), utf8: '4'
[14:     wl_keyboard] key: serial: 566; time: 300528; key: 83; state: 0 (released)
                      sym: KP_4         (65460), utf8: ''
```
xkbcli did have some more info on modifiers however:
```
default: keycode [ KP4  ] keysyms [ KP_4             ] unicode [ 4 ] layout [ English (US) (0) ] level [ 1 ] mods [ -Mod2 ] leds [ Num Lock ]
```

**Comment by ifreund** [comment]
This should be fixed by ec8f57e704f802eaa98275785f66618a7652fd11, thanks for the report!
